.clj-kondo/config.edn

```diff
@@ -1,20 +1 @@
-{:lint-as {manifold.utils/definterface+ clojure.core/definterface}
-
- :hooks {:analyze-call
-         {manifold.stream.core/def-source manifold.hooks/def-sink-or-source
-          manifold.stream.core/def-sink manifold.hooks/def-sink-or-source
-          manifold.stream.core/def-sink+source manifold.hooks/def-sink-or-source
-          manifold.deferred/both manifold.hooks/both
-          manifold.deferred/success-error-unrealized manifold.hooks/success-error-unrealized}}
-
-
-:config-in-call {manifold.stream.core/def-sink+source
-                 {:linters {:redefined-var {:level :off}}}
-
-                 manifold.stream.core/def-sink
-                 {:linters {:redefined-var {:level :off}}}
-
-                 manifold.stream.core/def-source
-                 {:linters {:redefined-var {:level :off}}}}
-
- :config-paths ["../resources/clj-kondo.exports/manifold/manifold/"]}
+{:config-paths ["../resources/clj-kondo.exports/manifold/manifold/"]}
```

Comment:
I think there is one `manifold/` directory too many?

Reply:
When clj-kondo is copying files out of the jar, it [looks like](https://github.com/clj-kondo/clj-kondo/blob/0a3f6837fa0964a1035cf8da05836bc8444d4279/src/clj_kondo/impl/core.clj#L220-L230) it doesn't really care what the subdirectories of `clj-kondo.exports/` are called, it just wants them to be two layers deep. I think it ignores the first layer ( `clj-kondo.exports/manifold` or `clj-kondo.exports/clj-commons` or whatever), and copies the second (`clj-kondo.exports/whatever/manifold`) into the project's `.clj-kondo/`, ending up with `.clj-kondo/manifold/config.edn`. I'm not exactly sure how it then later uses those copied configs, but things seem to work out OK.

Reply:
I love kondo, but the docs on this aspect have always been a bit confusing.

Reply:
`clj-kondo` [recommends this convention][docs]:

```
clj-kondo.exports/<your-org>/<your-libname>/
```

It seems reasonable to require a quasi-unique namespace for `<your-org>`, similar to maven dependencies (e.g. Clojars).

[docs]: https://github.com/clj-kondo/clj-kondo/blob/master/doc/config.md#exporting

Reply:
It's not that part, but the part about simultaneously using, and exporting, the same files.